funding.json

```diff
@@ -0,0 +1,98 @@
+        "name": "icalendar",     
+        "description": "The icalendar package is a RFC 5545 compatible parser/generator for iCalendar files.",     
+        "webpageUrl": {
+            "url": "https://icalendar.readthedocs.io/",
+            "wellKnown": "https://icalendar.readthedocs.io/.well-known/funding-manifest-urls"
+        },
+        "repositoryUrl": {
+            "url": "https://github.com/collective/icalendar/",
+            "wellKnown": "https://github.com/collective/icalendar/raw/main/.github/.well-known/funding-manifest-urls"
+        },
+        "licenses": ["spdx:BSD-2-Clause"],               
+        "tags": ["icalendar", "rfc5545", "parser", "generator", "programming", "python"]
+    }],
+
+    "funding": {
+        "channels": [{
+            "guid": "icalendar-open-source-collective",
+            "type": "payment-provider",
+            "address": "https://opencollective.com/python-icalendar",     
+            "description": "OpenCollective supports payment via credit card, Google Pay, or US bank ACH transfer." 
+        },
+        {
+            "guid": "github-sponsors-niccokunzmann",
+            "type": "payment-provider",
+            "address": "https://github.com/sponsors/niccokunzmann",
+            "description": "GitHub supports payment via credit card or invoice-based billing."
+        }],
+
+        "plans": [{
+            "guid": "fund-once-any-amount-usd",  
+            "status": "active",       
+            "name": "One-time donation",  
+            "description": "Choose any amount to give to icalendar.",    
+            "amount": 0,    
+            "currency": "USD", 
+            "frequency": "one-time", 
+            "channels": [
+                "icalendar-open-source-collective",
+                "github-sponsors-niccokunzmann"
+            ]
+        }, {
+            "guid": "fund-monthly-5-usd",  
+            "status": "active",       
+            "name": "Backer",  
+            "description": "Support icalendar with a monthly donation of $5 USD.",    
+            "amount": 5,    
+            "currency": "USD", 
+            "frequency": "monthly", 
+            "channels": [
+                "icalendar-open-source-collective",
+                "github-sponsors-niccokunzmann"
+            ]
+        }, {
+            "guid": "fund-monthly-100-usd",  
+            "status": "active",       
+            "name": "Sponsor",  
+            "description": "Support icalendar with a monthly donation of $100 USD.",    
+            "amount": 100,    
+            "currency": "USD", 
+            "frequency": "monthly", 
+            "channels": [
+                "icalendar-open-source-collective",
+                "github-sponsors-niccokunzmann"
+            ]
+        }],
+
+        "history": [{
+            "year": 2024,
+            "income": 202,
+            "currency": "USD"
+        }, {
+            "year": 2025,
+            "income": 12,
```

Comment:
Your GH sponsor profile shows you at $200/month. Perhaps GH Sponsors is for work outside of icalendar?

<img width="418" height="116" alt="Screenshot 2025-12-23 at 11 42 48 PM" src="https://github.com/user-attachments/assets/77875a49-2fea-4636-8ae6-bebc2557c9ad" />

Reply:
Removed github sponsors here: 9a4abf2